tests/service/data/SyntaxTree/Attribute/MultipleAttributes02.fs

```diff
@@ -0,0 +1,4 @@
+
+[<MyAttribute(foo ="bar")
+  MyAttribute(foo ="bar")>]
```

Comment:
Is it worth having a test where the second one is not indented far enough? Would it be considered offsides? What about a mixture of semicolons and newlines?